src/GenAPI/Microsoft.DotNet.GenAPI.Task/build/Microsoft.DotNet.GenAPI.Task.targets

```diff
@@ -49,9 +50,10 @@
       ExceptionMessage="$(GenAPIExceptionMessage)"
       ExcludeApiFiles="@(GenAPIExcludeApiList)"
       ExcludeAttributesFiles="@(GenAPIExcludeAttributesList)"
-      IncludeVisibleOutsideOfAssembly="$(GenAPIIncludeVisibleOutsideOfAssembly)"
+      RespectInternals="$(GenAPIRespectInternals)"
```

Comment:
Please, remember to update the SBRP generator.

Reply:
We don't use `GenAPIIncludeVisibleOutsideOfAssembly` in the SBRP generator.